courses/templates/course_detail.html

```diff
@@ -19,7 +19,7 @@
 {% endblock %}
 
 {% block content %}
-<div>
+<div class="parent-container">
```

Comment:
What's is this class here for?

Reply:
I will remove it, just added for future reference for adding scss for parent container but not require yet.

Reply:
First comment: if we are creating CSS classes ourselves, I would prefer that they be more descriptive than `parent-container`. Second comment: under what circumstances are you seeing a horizontal scroll? I'm not seeing it when I disable `overflow: hidden`. I would very strongly prefer that we introduce styles that prevent content overflow and scroll bars instead of just hiding it.